content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -53,24 +53,25 @@ creating a patch to fix it in the upstream project.
 
 ## Cloning the Kubernetes repository
 
-If you don't already have the kubernetes/kubernetes repository, get it now:
+Make sure your `kubernetes` fork is up-to-date with the `kubernetes/kubernetes` remote on
+GitHub (`master` branch), and clone your `kubernetes` fork.
 
 ```shell
 mkdir $GOPATH/src
 cd $GOPATH/src
-go get github.com/kubernetes/kubernetes
+git clone git@github.com:<your_github_username>/kubernetes.git
 ```
 
 Determine the base directory of your clone of the
 [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repository.
 For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes/kubernetes.`
+base directory is `$GOPATH/src/kubernetes.`
```

Comment:
This doesn't look right.

Reply:
@tengqm , can you mention what changes should i make?

Reply:
Okay, I see, you have change the command above from 'go get' to 'git clone'.
Thant makes the directory correct.